### PR TITLE
[TD-13722]<fix>: remove Requests (Inserts) panel in Requests row

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

**Review by afwerar — APPROVED** [review]
LGTM

**Review by zitsen — APPROVED** [review]
LGTM

File: dashboards/TDinsight.json [hunk 1]
```diff
@@ -2824,139 +2824,6 @@
       },
       "id": 108,
       "panels": [
```
Comment (zitsen) [inline dashboards/TDinsight.json:2826]: bump the version  in the end of the file.

    "version": 27

to next version:

    "version": 28

**Review by zitsen — CHANGES_REQUESTED** [review]
Bump the TDinsight version

**Comment by zitsen** [comment]
Better reduce the image file size with pngquant

File: dashboards/TDinsight.json (hunk 1 above)
Comment (huolibo) [reply dashboards/TDinsight.json:2826]: what is the meaning of schemaVersion? need to modify this to 28?

**Review by zitsen — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]

**Pull request merged**